File: WebAPI/Models/AuthoriseModel.cs
namespace WebAPI.Models
{
    public class AuthorizeModel
    {
        public string Name { get; set; }
        public string Email { get; set; }
        public bool IsAvailable { get; set; }

        public AuthorizeModel(string name, string email, bool isAvailable)
        {
            Name = name;
            Email = email;
            IsAvailable = isAvailable;
        }
    }
}

File: WebAPI/ClientApp/src/app/models/AuthoriseModel.ts
export class AuthoriseModel {
    public name: string;
    public email: string;
    public isAvailable: boolean;

    public constructor(_name: string, _email: string, _isAvailable: boolean) {
        this.name = _name;
        this.email = _email;
        this.isAvailable = _isAvailable;
    }
}